views/index.jade

```diff
@@ -14,7 +15,7 @@ block content
           input#email.form-control(type='email', placeholder='Enter Email', name='email', required='')
         .form-group
           label.control-label(for='git_url') GITURL:
-          input#git_url.form-control(type='text', placeholder='Enter URL of Github Repository', name='git_url')
+          select#git_url.form-control(type='text', placeholder='Enter URL of Github Repository', name='git_url')
```

Comment:
What is the use of select here if we have no option?

@pri22296  Editable select was use there because we had some predefined static values. For this, we have to fetch data from an API. These two are different and hence i think this change is unnecessary